src/www/ui_components/server_list.ts

```diff
@@ -43,19 +50,21 @@ export class ServerList extends PolymerElement {
         }
       </style>
 
-      <template is="dom-repeat" items="[[servers]]">
-        <server-card 
-          disabled="[[item.errorMessageId]]" 
-          error-message="[[localize(item.errorMessageId)]]" 
-          expanded="[[hasSingleServer]]"
-          localize="[[localize]]" 
-          root-path="[[rootPath]]" 
-          server-address="[[item.address]]" 
-          server-id="[[item.id]]"
-          server-name="[[item.name]]" 
-          is-outline-server="[[item.isOutlineServer]]"
-        ></server-card>
-      </template>
+      <div id="sortableContainer">
```

Comment:
digging into the source code, SortableJS seems to not support dragging across multiple nested web components, so I opted to create a container here